### DISCUSSION TITLE
你的牙齿很值钱 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#52] [Announcements]
# 你的牙齿很值钱 - forecho's Blog

引言
事情的开始是这样的，那天下班在家上网逛 V2EX，然后看到了这篇文章《我看牙医的经历》，然后联想到我的两颗烂牙，想想就觉得后怕，于是下定决心去补牙，不然最后只会越拖越严重。
选择哪里看牙
说道看牙，我的第一反应并没有说是去正规医院看病，因为我对他们也不了解，我怕被坑，然后正好兔子公司的有福利，可以用 Q点 兑换瑞尔齿科的金卡，洗牙免费，看牙打折，网上简单搜索了一下评价还可以，于是我们就打打电话预约时间，先洗牙，顺便检查牙齿。

https://blog.forecho.com/your-teeth-are-very-valuable.html

**Comment by dox-xu** [comment]
牙齿健康确实很重要，可惜老一辈的人很多都没有口腔保健意识
